.github/workflows/build-zip-lint.yml

```
@@ -0,0 +1,51 @@
+  BuildFolderName: CrcLauncher
+  FrameworkVersion: net8.0-windows10.0.22621.0
+
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Set up .NET
+        uses: actions/setup-dotnet@v2
```

actionlint (1.7.4)

the runner of "actions/setup-dotnet@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)